### DISCUSSION TITLE
AAR for react-native-reanimated has been found /Users/zimdev/work/ZimMobile/node_modules/react-native-reanimated/android/react-native-reanimated-68-jsc.aar

### DISCUSSION THREAD

**Discussion by MacDinhThanh** [software-mansion/react-native-reanimated#3787] [Q&A]
AAR for react-native-reanimated has been found
/Users/zimdev/work/ZimMobile/node_modules/react-native-reanimated/android/react-native-reanimated-68-jsc.aar

**Comment by tomekzaw** [comment]
Hey, this is not an error. This is just a message that states that Reanimated doesn't need to be built from sources as AAR has been successfully found.
